po/de.po

```diff
@@ -130,8 +130,9 @@ msgstr "Der Ausdruck %s ist falsch."
 
 #: src/string/i18n.c:15
 #, c-format
-msgid "The expression (as strings) %s is false."
-msgstr "Der Ausdruck (als Zeichenkette) %s ist falsch."
+msgid "The expression (as strings) %s is false: actual=`%s` expected=`%s`."
+msgstr ""
+"Der Ausdruck (als Zeichenkette) %s ist falsch: actual=`%s` expected=`%s`."
```

Comment:
Yes, I know. 
I thought about the "actual" string as the current string fed into the test.
As you said "aktuell" is used in conjunction with time. So if we take the develop process as a timeline this should be correct.
But I got your point now. I dont know which version is the better one.